### PR TITLE
[zh-cn]sync security/_index.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/security/_index.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/_index.md [hunk 1]
```diff
@@ -2,4 +2,253 @@
...
+
+Kubernetes is based on a cloud-native architecture, and draws on advice from the
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}} about good practice for
+cloud native information security.
+-->
+Kubernetes 文档的这一部分内容的旨在引导你学习如何更安全地运行工作负载，
+以及维护 Kubernetes 集群的基本安全性。
+
+Kubernetes 基于云原生架构，并借鉴了
+{{<lossary_tooltip text="CNCF" term_id="cncf" >}} 有关云原生信息安全良好实践的建议。
```
Comment (dipesh-rawat) [inline content/zh-cn/docs/concepts/security/_index.md:29]: Typo in the `glossary_tooltip` shortcode name is leading to [build failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65d0b4b1962ad40008cab067).

```suggestion
{{< glossary_tooltip text="CNCF" term_id="cncf" >}} 有关云原生信息安全良好实践的建议。
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 2]
```diff
@@ -2,4 +2,253 @@
...
+
+<!--
+## Kubernetes security mechanisms {#security-mechanisms}
+
+Kubernetes includes several APIs and security controls, as well as ways to
+define [policies](#policies) that can form part of how you manage information security.
+-->
+## Kubernetes 安全机制 {#security-mechanisms}
+
+Kubernetes 包含多个 API 和安全控制，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:47]:
```suggestion
Kubernetes 包含多个 API 和安全组件，
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 3]
```diff
@@ -2,4 +2,253 @@
...
+以及定义[策略](#policies)的方法，这些策略可以作为你管理信息安全的一部分。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:48]:
```suggestion
以及定义[策略](#policies)的方法，这些策略可以作为你的信息安全管理的一部分。
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 4]
```diff
@@ -2,4 +2,253 @@
...
+<!--
+Kubernetes expects you to configure and use TLS to provide
+[data encryption in transit](/docs/tasks/tls/managing-tls-in-a-cluster/)
+within the control plane, and between the control plane and its clients.
+You can also enable [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)
+for the data stored within Kubernetes control plane; this is separate from using
+encryption at rest for your own workloads' data, which might also be a good idea.
+-->
+Kubernetes 希望你配置和使用 TLS
+在控制平面内以及控制平面与其客户端之间提供[传输中的数据加密](/zh-cn/docs/tasks/tls/managing-tls-in-a-cluster/)。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:69]:
```suggestion
Kubernetes 希望你配置并使用 TLS，
以便在控制平面内以及控制平面与其客户端之间提供[传输中的数据加密](/zh-cn/docs/tasks/tls/managing-tls-in-a-cluster/)。
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 5]
```diff
@@ -2,4 +2,253 @@
...
+### Workload protection
+
+Enforce [Pod security standards](/docs/concepts/security/pod-security-standards/) to
+ensure that Pods and their containers are isolated appropriately. You can also use
+[RuntimeClasses](/docs/concepts/containers/runtime-class) to define custom isolation
+if you need it.
+-->
+### 工具负载保护
+
+执行 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)以确保
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:94]:
```suggestion
实施 [Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)以确保
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 6]
```diff
@@ -2,4 +2,253 @@
...
+
+Kubernetes [audit logging](/docs/tasks/debug/debug-cluster/audit/) provides a
+security-relevant, chronological set of records documenting the sequence of actions
+in a cluster. The cluster audits the activities generated by users, by applications
+that use the Kubernetes API, and by the control plane itself.
+-->
+### 审计
+
+Kubernetes [审计日志记录](/zh-cn/docs/tasks/debug/debug-cluster/audit/)提供了一组与安全相关、
+按时间顺序排列的记录，记录了集群中的操作顺序。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:119]:
```suggestion
按时间顺序排列的记录，记录了集群中的操作序列。
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 7]
```diff
@@ -2,4 +2,253 @@
...
+
+If you are running a Kubernetes cluster on your own hardware or a different cloud provider,
+consult your documentation for security best practices.
+Here are links to some of the popular cloud providers' security documentation:
+-->
+## 云提供商安全
+
+{{% thirdparty-content vendor="true" %}}
+
+如果你在自己的硬件或不同的云提供商上运行 Kubernetes 集群，请参阅云提供商的文档以了解安全最佳实践。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:135]:
```suggestion
如果你在自己的硬件或不同的云平台上运行 Kubernetes 集群，请参阅对应云平台的文档以了解安全最佳实践。
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 8]
```diff
@@ -2,4 +2,253 @@
...
+（对某人可以使用 Kubernetes API 进行哪些更改的声明性限制）。
+
+<!--
+However, you can also rely on policy implementations from the wider
+ecosystem around Kubernetes. Kubernetes provides extension mechanisms
+to let those ecosystem projects implement their own policy controls
+on source code review, container image approval, API access controls,
+networking, and more.
+-->
+你还可以依赖 Kubernetes 周边更广泛的生态系统的策略实施。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:192]:
```suggestion
你还可以依赖 Kubernetes 周边更广泛的生态系统的策略实现。
```

File: content/zh-cn/docs/concepts/security/_index.md [hunk 9]
```diff
@@ -2,4 +2,253 @@
...
+-->
+了解相关的 Kubernetes 安全主题：
+
+* [保护集群](/zh-cn/docs/tasks/administer-cluster/secure-a-cluster/)
+* Kubernetes 中的[已知漏洞](/zh-cn/docs/reference/issues-security/official-cve-feed/)（以及更多信息的链接）
+* [传输中的数据加密](/zh-cn/docs/tasks/tls/managing-tls-in-a-cluster/)（针对控制平面）
+* [静态数据加密](/zh-cn/docs/tasks/administer-cluster/encrypt-data/)
+* [控制对 Kubernetes API 的访问](/zh-cn/docs/concepts/security/controlling-access)
+* Pod 的 [网络策略](/zh-cn/docs/concepts/services-networking/network-policies/)
+* [Kubernetes 中的秘密](/zh-cn/docs/concepts/configuration/secret/)
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/_index.md:226]:
```suggestion
* [Kubernetes 中的 Secret](/zh-cn/docs/concepts/configuration/secret/)
```

**Pull request merged**